### PR TITLE
Compatible old ReportUtil usage and supply test cases 

### PR DESCRIPTION
**Pull request by guanchao-yang**
### Motivation

Before some alipay users use `ReportUtil` with the full package `com.alipay.sofa.common.log.ReportUtil` but in this version `ReportUtil` has been moved to `com.alipay.sofa.common.utils.ReportUtil`. Consider many users has used, so we should be compatible with it

### Modification

`com.alipay.sofa.common.log.ReportUtil`  extends `com.alipay.sofa.common.utils.ReportUtil` for compatible

### Result

Fixes

 #14 

#16 only litte cases and after should provide more test cases

### CODE REVIEW

**Comment by khotyn** [comment]
Please attach this PR to a milestone, and the PR description should be have more detail like [this](https://github.com/alipay/sofa-rpc/blob/master/.github/PULL_REQUEST_TEMPLATE.md)

File: .github/ISSUE_TEMPLATE/Ask_Question.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+describe your use scenes (why need this feature)
+
+### Your advice
+
+describe the advice or solution you'd like
+
+### Environment
+
+- SOFA Lookout version:
```
Comment (khotyn) [inline .github/ISSUE_TEMPLATE/Ask_Question.md]: DO NOT JUST COPY FROM OTHER PROJECT!

File: src/test/java/com/alipay/sofa/common/utils/AssertUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package com.alipay.sofa.common.utils;
```
Comment (khotyn) [inline src/test/java/com/alipay/sofa/common/utils/AssertUtilTest.java]: What is the purpose of this test case? It seems that it is not related to this issue.

Comment (guanchao-yang) [reply src/test/java/com/alipay/sofa/common/utils/AssertUtilTest.java]: #16 only litte cases and after should provide more test cases

Comment (khotyn) [reply src/test/java/com/alipay/sofa/common/utils/AssertUtilTest.java]: Please only do one thing in on PR next time.

File: src/main/java/com/alipay/sofa/common/log/ReportUtil.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+package com.alipay.sofa.common.log;
+
+/**
+ * ReportUtil Compatible history usage
+ * <p>
+ * Output logs to console and default logger
+ * <p>
+ * Created by yangguanchao on 18/06/04.
+ */
+public class ReportUtil extends com.alipay.sofa.common.utils.ReportUtil{
```
Comment (khotyn) [inline src/main/java/com/alipay/sofa/common/log/ReportUtil.java]: Add a Deprecated annotation here cause it is only for the compatible problem.

File: src/test/java/com/alipay/sofa/common/utils/AssertUtilTest.java (hunk 2 above)
Comment (guanchao-yang) [reply src/test/java/com/alipay/sofa/common/utils/AssertUtilTest.java]: ok

**Comment by guanchao-yang** [comment]
@khotyn
